script.js

```diff
@@ -1,35 +1,108 @@
+				minutes: 0,
+				seconds: 0,
+				miliseconds: 0
+			}
+		};
+		this.start = this.start.bind(this);
+		this.stop = this.stop.bind(this);
+		this.reset= this.reset.bind(this);
+		this.saveResult=this.saveResult.bind(this);
+	}
+	
+	print() {
+	//		console.log(this.format(this.state.times));	
+		return	this.innerText = this.format(this.state.times);
+	}
 
-const string1 = 'Hello';
-const string2 =  'World';
-function helloWorld(string1, string2){
-	console.log(`${string1} ${string2}`);
-}
-helloWorld(string1, string2);
+	format(times) {
+		return `${pad0(times.minutes)} : ${pad0(times.seconds)} : ${pad0(Math.floor(times.miliseconds))}`;
+	}
 
-// Zadanie 2
+	reset() {
+		this.setState({
+			times : {
+				minutes: 0,
+				seconds: 0,
+				miliseconds: 0
+			}
+		})
+	}
 
-// function multiply (a=1, b=1){return a*b}
-const multiply = (a=1, b=1) => a*b
+	start() {
+		if (!this.state.running){
+			this.setState({
+				running : true
+			});
+			this.watch = setInterval(() => this.step(), 10);
+		}
+	}
 
-console.log(multiply(5,4));
-console.log(multiply(5));
+	step() {
+		//console.log(this.state);
+		if (!this.state.running) return;
+		this.calculate();
+		this.print();
+	}
 
-// Zadanie 3
+	calculate() {
+		//console.log(this.state.times);
+		this.setState({times: {minutes: this.state.times.minutes, seconds: this.state.times.seconds, miliseconds : this.state.times.miliseconds +=1} })
+		if(this.state.times.miliseconds >= 100) {
+			this.setState({times: {minutes: this.state.times.minutes, seconds: this.state.times.seconds += 1, miliseconds: this.state.times.miliseconds = 0} })
+		}
+		//console.log(this.state.times);
+		if (this.state.times.seconds >= 60) {
+			this.setState({times: { minutes: this.state.times.minutes += 1, seconds:this.state.times.seconds = 0, miliseconds: this.state.times.miliseconds}})
+		}
+	}
 
-const avarage = (...arg) => arg.reduce((a,b) => a+b)/arg.length;  
-console.log(avarage(2));
-console.log(avarage(5,10,20));
+	stop() {
+		this.setState({running : false})
+		clearInterval(this.watch);
+	}
 
-//Zadanie 4
+	saveResult(){ 
+		const newResult = document.createElement("li");
+		newResult.innerHTML = this.innerText;
+		resultsList.appendChild(newResult);
+	}
 
-const grades = [1, 5, 5, 5, 4, 3, 3, 2, 1];
-console.log(avarage(...grades)); 
+	resetResultList(){
+	 resultsList.innerHTML = "";
+	}
 
-//Zadanie5
+	render(){ 
+		return(
+			React.createElement('div',{},
+				React.createElement('button',{onClick: this.start},'Start'),
+				React.createElement('button',{onClick: this.stop},'Stop'),
+				React.createElement('button',{onClick: this.reset},'Reset'),
+				React.createElement('div',{},
+					React.createElement('p',{}, this.print())
+				),
+				React.createElement('h1',{},'Lista Wyników'),
+				React.createElement('ul',{id:"resultsList"},),
+				React.createElement('button',{onClick: this.resetResultList}, 'Zresetuj listę'),
+				React.createElement('button',{onClick: this.saveResult},'Zapisz wynik')
+			)
```

Comment:
ale sobie utrudniłeś życie - w tamtym module przerabialiśmy JSX, który jest dużo przyjemniejszy, a tu z niego nie korzystasz. Zainstaluj sobie odpowiedni preset `babel-preset-react`, dodaj "react" do babelrc presetów i przerób to na JSX